site/app/views/grading/ElectronicGraderView.php

```diff
@@ -494,47 +494,55 @@ function sortTable(sort_element_index){
      * @param bool $show_import_teams_button
      * @param bool $show_export_teams_button
      * @param bool $show_edit_teams
+     * @param string $past_grade_start_date
+     * @param bool $view_all
+     * @param string $sort
+     * @param string $direction
+     * @param bool $anon_mode
+     * @param array<string, bool> $overrides
+     * @param array<string, string> $anon_ids
+     * @param bool $inquiry_status
+     * @param array<string> $grading_details_columns
      * @return string
      */
-    public function detailsPage(Gradeable $gradeable, $graded_gradeables, $teamless_users, $graders, $empty_teams, $show_all_sections_button, $show_import_teams_button, $show_export_teams_button, $show_edit_teams, $past_grade_start_date, $view_all, $sort, $direction, $anon_mode, $overrides, $anon_ids, $inquiry_status) {
+    public function detailsPage(Gradeable $gradeable, $graded_gradeables, $teamless_users, $graders, $empty_teams, $show_all_sections_button, $show_import_teams_button, $show_export_teams_button, $show_edit_teams, $past_grade_start_date, $view_all, $sort, $direction, $anon_mode, $overrides, $anon_ids, $inquiry_status, $grading_details_columns) {
```

Comment:
You could strengthen the constraints here by adding a native type annotation to the new `$grading_details_columns` parameter.